File: 백준/Bronze/9063. 대지/대지.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int T = sc.nextInt();

        int[][] point = new int[T][2];

        for (int i = 0; i < T; i++) {
            point[i][0] = sc.nextInt();
            point[i][1] = sc.nextInt();
        }

        int minX = Integer.MAX_VALUE;
        int maxX = Integer.MIN_VALUE;

        int minY = Integer.MAX_VALUE;
        int maxY = Integer.MIN_VALUE;

        for (int i = 0; i < T; i++) {
            minX = Math.min(point[i][0], minX);
            minY = Math.min(point[i][1], minY);

            maxX = Math.max(point[i][0], maxX);
            maxY = Math.max(point[i][1], maxY);
        }

        System.out.println((maxX - minX) * (maxY - minY));
    }
}

File: 백준/Bronze/9063. 대지/대지.py
import sys

# sys.stdin = open("input.txt", "r")

n = int(sys.stdin.readline().strip())

max_x = max_y = -10000
min_x = min_y = 10000

for _ in range(n):
    x, y = map(int, sys.stdin.readline().split())

    max_x = max(max_x, x)
    min_x = min(min_x, x)

    max_y = max(max_y, y)
    min_y = min(min_y, y)

print(abs(max_x - min_x) * abs(max_y - min_y))